PowerVCF.psm1

```diff
@@ -6330,24 +6330,45 @@ Export-ModuleMember -Function Debug-CatchWriter
 
 
 #Region JSON Export Functions
+
 Function Export-VCFManagementDomainJsonSpec {
+    <#
+        .SYNOPSIS
+        Generates a JSON file from the VMware Cloud Foundation Planning & Preparation Workbook to perform a VMware Cloud
+        Foundation Bring-up. Requires the installation of the ImportExcel Powershell module.
+
+        .DESCRIPTION
+        The Export-VCFManagementDomainJsonSpec cmdlet creates the JSON specification file using the Planning and Preparation workbook
+        to deploy a management domain.
+
+        .EXAMPLE
+        Export-VCFManagementDomainJsonSpec -workbook .\pnp-workbook.xlsx -jsonPath .\domainSpec.json
+        This example creates a JSON deployment specification for a Management Domain using the Planning and Preparation Workbook.
+
+        .PARAMETER workbook
+        The path to the Planning and Preparation Workbook (.xlsx) file.
+
+        .PARAMETER jsonFile
+        The fully qualified path to the JSON specification file.
+    #>
 
     Param (
         [Parameter (Mandatory = $true)] [String]$workbook,
         [Parameter (Mandatory = $true)] [String]$jsonPath
     )
+
     # Confirm the presence of ImportExcel module
-    if (!(Get-InstalledModule -name "ImportExcel"  -MinimumVersion 7.8.5 -ErrorAction SilentlyContinue)) {
```

Comment:
Why don't use on psd1 requiresmodule ?

Reply:
That would require everyone who installs PowerVCF to have ImportExcel installed. Not sure we want to enforce that as not everyone will use the export json functions